From 2015b643c95127fc34b05e3252562599ce485b14 Mon Sep 17 00:00:00 2001
From: Michael Webster <miketwebster@gmail.com>
Date: Sat, 3 Jun 2023 09:38:22 -0400
Subject: [PATCH] xapp-debug.c: build only with ENABLE_DEBUG.

xapp-debug.h gets disable, so the build would break when buildtype is
'release'.

Fixes #157.
---
 libxapp/xapp-debug.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/libxapp/xapp-debug.c b/libxapp/xapp-debug.c
index 8bb10ea..da2b913 100644
--- a/libxapp/xapp-debug.c
+++ b/libxapp/xapp-debug.c
@@ -28,6 +28,8 @@
 
 #include "xapp-debug.h"
 
+#ifdef ENABLE_DEBUG
+
 static DebugFlags flags = 0;
 static gboolean initialized = FALSE;
 
@@ -127,3 +129,4 @@ xapp_debug_valist (DebugFlags flag,
   }
 }
 
+#endif /* ENABLE_DEBUG */
